src/schema.graphql

```diff
@@ -46,6 +46,7 @@ type ArenaTopic {
 
 type ArenaUser {
   displayName: String!
+  groupTitleArray: [String]
```

Comment:
Stusser litt på typen her. I TS-land vil dette være en (string | undefined)[] | undefined

Reply:
Jeg er usikker om alle brukere kommer til å ha en groupTitleArray eller ikke. Burde det da være [String]! for å gjøre den til (string | undefined)[] eller er det bestemt at alle skal ha det?

Reply:
Her må du nok nesten lese nodebb-docsa. Men jeg vil regne med at en mer naturlig type er enten `string[]` eller `string[] | undefined`. Kan ikke se for meg at de returnerer et array med undefined i.